app/dataSources/validationPatterns.tsx

```diff
@@ -1,16 +1,16 @@
 import * as z from "zod/v4";
 
 const validationPattern = {
-    wikiPageName: {
+    wikiPageUrl: {
         zod: z.string().regex(/^[a-zA-Z()_0-9]+$/),
         string: "^[a-zA-Z()_0-9]+$"
     },
-    googleSheetsUrl: {
+    googleSheetUrl: {
         zod: z.url({
             protocol: /^https$/,
             hostname: z.regexes.domain
         }),
-        string: "^https://([a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?\\.)+[a-zA-Z]{2,}$"
+        string: "https://.*" // eslint-disable-line no-useless-escape
```

Comment:
This is my exact concern with these drifting a part from one another 😬

Reply:
Actually, this can be changed back. Originally this was throwing an error, but it's now fixed.